bog_app/app/controllers/creatures_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CreaturesController < ApplicationController
+
+
+  def index
+    @creatures = Creature.all
+    # if params[:search]
+    #   @Creatures = Creature.search(params[:search]).order("created_at DESC")
```

Comment:
On lines 7 and 9, you've capitalized @Creatures. This might be part of the issue that you're having with the search functionality.